platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionDefinition.java

```diff
@@ -48,7 +48,17 @@
   float getPixelRatio();
 
   /**
-   * Gest the type of the OfflineRegionDefinition for telemetry ("tileregion", "shaperegion").
+   Specifies whether to include ideographic characters in downloaded font data.
+   Ideographic characters make up the majority of downloaded font data, but
+   it is possible to configure the renderer to use locally installed fonts
+   instead of relying on fonts downloaded as part of the offline pack.
+
+   Defaults to `true`
```

Comment:
Could you add a `@return` prefix here describing briefly the returned variable?

Reply:
:+1: